### ISSUE TITLE
Parsing podcast series and podcast episodes

### ISSUE THREAD

**Issue by Susexe** [cinemagoer/cinemagoerng#3]
Currently, it fails. I suppose you didn't implement this yet. I'm not sure how much it differs from other title types.

**Comment by uyar** [comment]
Yes, I've noticed them. It could be as easy as adding two new dataclasses with the correct with the correct type name values. But I have to check if they differ in other ways and also need to think about all the test cases.
